### ISSUE TITLE
AutoGTP should be more verbose

### ISSUE THREAD

**Issue by Nazgand** [leela-zero/leela-zero#1534]
During the first game with a new network, it looks like AutoGTP has crashed even it if has not. Today, I killed AutoGTP and restarted it, so I suggest something along the lines of `Downloading new network` or `Training new network` should be output so that AutoGTP is not unnecessarily killed by newbs like me.

```
7 game(s) (7 self played and 0 matches) played in 265 minutes = 2278 seconds/game, 8366 ms/move, last game took 1030 seconds.
{
    "cmd": "selfplay",
    "hash": "d01879964d578b676714251164f7289da023f14c0063b1721e5e3cd2e8d51ae0",
    "minimum_autogtp_version": "16",
    "minimum_leelaz_version": "0.15",
    "options": {
        "noise": "true",
        "playouts": "0",
        "randomcnt": "999",
        "resignation_percent": "5",
        "visits": "3201"
    },
    "options_hash": "df715e",
    "random_seed": "6566234552993981740",
    "required_client_version": "16"
}

Got new job: selfplay
^X^D
PS C:\Users\Nazgand\Downloads\leela-zero-0.15-win64> .\autogtp.exe -k sgf
AutoGTP v16
Using 1 thread(s) for GPU(s).
Starting tuning process, please wait...
Net filename: networks/d01879964d578b676714251164f7289da023f14c0063b1721e5e3cd2e8d51ae0
net: d01879964d578b676714251164f7289da023f14c0063b1721e5e3cd2e8d51ae0.
./leelaz --tune-only -w networks/d01879964d578b676714251164f7289da023f14c0063b1721e5e3cd2e8d51ae0
```

**Comment by Nazgand** [comment]
> you should be patient

I plan to be patient next time this happens because I now know the reason for the lag. My suggestion still stands.

**Comment by l1t1** [comment]
wget -c http://zero.sjeng.org/networks/d01879964d578b676714251164f7289da023f1e5e3cd2e8d51ae0.gz
41%[++++++++++++++++++++                              ]15.73M  0.91 B/s    eta 675d

**Comment by Nazgand** [comment]
Perhaps it is because I am using Windows at the moment (Fedora refuses to boot properly). I did not see anything like that. The first mention of d01879964d578b676714251164f7289da023f14c0063b1721e5e3cd2e8d51ae0 is in the excerpt I posted.
`Got new job: selfplay` followed by no change for a while is not as good as an explanation about the new network needing downloading and tuning or whatever it is doing.
